onnxruntime/contrib_ops/rocm/fused_conv.cc

```diff
@@ -271,38 +278,19 @@ class FusedConv : public onnxruntime::rocm::Conv<T, false> {
     miopenFusionOpDescriptor_t bias_b_op = nullptr;
     miopenFusionOpDescriptor_t bias_z_op = nullptr;
     miopenFusionOpDescriptor_t act_op = nullptr;
-    miopenOperatorArgs_t fusion_args = nullptr;
 
     // TODO: There is a potential problem. miopenHandle_t may be destroyed and
     //       re-created later, sharing the same address. Currently there is any way
     //       to detect it?
     mutable std::unordered_set<miopenHandle_t> compiled_on;
 
-    FusedConvFusionData(const FusedConvFusionData&) = delete;
-    FusedConvFusionData& operator=(const FusedConvFusionData&) = delete;
-
-    FusedConvFusionData(FusedConvFusionData&& other) {
-      *this = std::move(other);
-    }
-    FusedConvFusionData& operator=(FusedConvFusionData&& other) {
-      std::swap(this->plan, other.plan);
-      std::swap(this->fusion_args, other.fusion_args);
-      this->conv_op = other.conv_op;
-      this->bias_b_op = other.bias_b_op;
-      this->bias_z_op = other.bias_z_op;
-      this->act_op = other.act_op;
-      this->compiled_on = std::move(other.compiled_on);
-      return *this;
-    }
+    ORT_DISALLOW_COPY_ASSIGNMENT_AND_MOVE(FusedConvFusionData);
```

Comment:
Are the comments above (in lines 269 - 274) still relevant? Doesn't this change ensure that a fusion descriptor will never be shared my multiple `FusedConv` objects? If so, please update the comments.

Reply:
fusion plan is still shared because the cache still exists, but fusion args will not be shared, and the fusion args must not be shared, this the purpose of this PR.